modules/table/src/main/java/org/apache/ignite/internal/utils/RebalanceUtil.java

```diff
@@ -157,6 +171,12 @@ public class RebalanceUtil {
     /** Key prefix for stable assignments. */
     public static final String STABLE_ASSIGNMENTS_PREFIX = "assignments.stable.";
 
+    /** Key prefix for switch reduce assignments. */
+    public static final String ASSIGNMENTS_SWITCH_REDUCE_PREFIX = "assignments.switch.reduce.";
```

Comment:
I guess it's not my business, but these names are too long. We will pollute metastorage with key names, it's horrible. I'm not laming you, of course

Reply:
It's possible to change values to something like "a.s.r", wdyt about it?

Reply:
We can do it later, because it is unrelated. It's just funny how we compare "writeObject" with native UOS, but then send these big ass strings over the network